.github/workflows/dci-merge.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+name: DCI
```

Comment:
I think we should name this DCI Merge, so we can see unique workflow names in the checks:
https://github.com/redhatci/ansible-collection-redhatci-ocp

Reply:
no we cannot at it needs to be the same name to satisfy the required jobs.